packages/native/src/index.js

```diff
@@ -5,46 +5,7 @@ import { styled } from './base'
 
 const css = createCss(reactNative.StyleSheet)
 
-const components = [
-  'ActivityIndicator',
-  'Button',
-  'DatePickerIOS',
-  'DrawerLayoutAndroid',
-  'FlatList',
-  'Image',
-  'ImageBackground',
-  'KeyboardAvoidingView',
-  'ListView',
-  'Modal',
-  'NavigatorIOS',
-  'Picker',
-  'PickerIOS',
-  'Pressable',
-  'ProgressBarAndroid',
-  'ProgressViewIOS',
-  'RecyclerViewBackedScrollView',
-  'RefreshControl',
-  'SafeAreaView',
-  'ScrollView',
-  'SectionList',
-  'SegmentedControlIOS',
-  'Slider',
-  'SnapshotViewIOS',
-  'StatusBar',
-  'SwipeableListView',
-  'Switch',
-  'SwitchIOS',
-  'TabBarIOS',
-  'Text',
-  'TextInput',
-  'ToolbarAndroid',
-  'TouchableHighlight',
-  'TouchableNativeFeedback',
-  'TouchableOpacity',
-  'TouchableWithoutFeedback',
-  'View',
-  'ViewPagerAndroid'
-]
+const components = Object.keys(reactNative)
```

Comment:
Expecting all of those to be components seems to be rather inaccurate/not future-proof, isnt it?

Reply:
My initial thought was maybe it could be done that way just for the sake of typing since majority of user who use styled.* would probably only do it to those component they know - but after thinking a bit, it'd be confusing for new developer. The new iteration should address this